src/usr/share/netcheck/netcheck-daemon.py

```diff
@@ -96,19 +96,22 @@ def sig_term_handler(signal, stack_frame):
     daemon_context = daemon.DaemonContext(
         working_directory='/',
         pidfile=pidlockfile.PIDLockFile(PID_FILE),
-        umask=0
+        # TODO: Set this back to 0 when NetCheck properly drops permissions.
+        umask=0o022
```

Comment:
This should **not** eventually be 0. Zero is the most permissive. Define the umask at the top of the file like I do in the other daemon code:
PROGRAM_UMASK = 0o027  # -rw-r----- and drwxr-x---